malevo/swagger.yml

```diff
@@ -0,0 +1,174 @@
+        - name: epoch
+          in: query
+          description: Selected epoch id
+          required: true
+          type: integer
+        - name: groundTruthClassId
+          in: query
+          description: Ground truth class id
+          required: true
+          type: integer
+        - name: predictedClassId
+          in: query
+          description: Predicted class id
+          required: true
+          type: integer
+        - name: numCount
+          in: query
+          description: Number of images
+          required: false
+          default: 100
+          minimum: 1
+          type: integer
+      responses:
+        '200':
+          description: >-
+            Successful operation returns a list of image ids
+          schema:
+            type: array
+            items:
+              type: integer
+        '400':
+          description: Invalid input value
+
+  /class/crossEntropy:
+    get:
+      tags:
+        - class
+      summary: Cross-entropy values
+      description: >-
+        Returns the cross-entropy values for a list of given image ids and epochs
+      produces:
+        - application/json
+      parameters:
+        - name: dataset
+          in: query
+          description: Phovea dataset id
+          required: true
+          type: string
+        - name: epochs
+          in: query
+          description: List of epochs
+          required: true
+          type: array
+          items:
+            type: integer
+        - name: imageIds
+          in: query
+          description: List of image ids
+          required: true
+          type: array
+          items:
+            type: integer
+      responses:
+        '200':
+          description: >-
+            Successful operation returns a matrix of cross-entropy values
+            with image ids as rows and epochs as columns.
+          schema:
+            type: array
+            items:
+              type: array
+              items:
+                type: integer
+        '400':
+          description: Invalid input value
+
+  /images/imageSprite:
+    get:
+      tags:
+        - images
+      summary: Generate an image sprite
+      description: >-
+        Returns an image sprite for all given image ids
+      produces:
+        - image/png
+      parameters:
+        - name: dataset
+          in: query
+          description: Phovea dataset id
+          required: true
+          type: string
+        - name: imageIds
+          in: query
+          description: List of image ids
+          required: true
+          type: array
+          items:
+            type: integer
+      responses:
+        '200':
+          description: Successful operation returns an image sprite
+          schema:
+            type: file
+          headers:
+            Content-type:
+              type: string
+              description: image/png; charset=utf-8
+        '400':
+          description: Invalid input value
+
+  /images/softmaxSprite:
```

Comment:
shouldn't a softmax sprite contain the softmax values for an image over several epochs? We're missing the epoch ids in the call